documentation/get-started/concepts.en.md

```diff
@@ -1,46 +1,37 @@
 ---
-title: Concepts
+title: Principles
 tx_slug: documentation_get-started_concepts
 ---
 
-# Concepts
+# Principles
 
-QField was designed with a few key concepts in mind.
+QField is designed with a few key principles in mind.
 
-## Keep it simple
 
-The requirements on the field are not the same as on a desktop. The
-screen is smaller, the input devices are different and the tasks are
-different.
+## Keeping it simple
 
-QField aims to help users to perform the tasks they need to do without
-cluttering the user interface. This means, that only tasks which need to
-be done on the field are availble from the interface. Everything else is
-not.
+The requirements in the field are not the same as on a desktop.
+Firstly, when using a phone or a tablet, the canvas to work on is much more limited than a computer monitor.
+Secondly, devices for data collection as well as the individual tasks that need to be performed are vastly different.
 
-This means that everything like layer styling, form definitions and
-other project setup steps should be done on a computer with QGIS
-installed first.
+QField aims to help users do fieldwork without the nuisance of a cluttered user interface.
+In other words, only the relevant parts of a task are accessible on the interface, while less important things remain out of sight.
 
-## Be compatible with QGIS
+Before heading out to the field though, preliminary steps such as layer styling, tailored forms, or any other project configurations should first be done on the QGIS desktop version.
```

Comment:
```suggestion
Before heading out to the field though, preliminary steps such as layer styling, tailored forms, input validation, or any other project configurations should first be done on the desktop via QGIS.
```

I guess the original sentence here was from very very long time ago.

Reply:
Yes, agree, no implications made.